Sources/HttpServer/Responses/ResponseBuilder.swift

```diff
@@ -38,11 +38,13 @@ public class ResponseBuilder {
                 body: Data(body.utf8))
     }
 
-    public func generateFile(body: Data?) -> HttpResponse {
-        return HttpResponse(statusCode: 200,
-                statusPhrase: "OK",
-                headers: ["Content-Length":String(body!.count), "Content-Type":"text/html"],
-                body: body!)
+    public func generate200ResponseWithFileContent(content: Data?, contentType: (fileType: String, fileExt: String)) -> HttpResponse {
+        self.resetBuilder()
```

Comment:
was this function working previously without `self.resetBuilder`? it looks like some of the generator functions are resetting and some are not.

Reply:
I gradually put the reset() in all of them. I ran into some problem when they don't get reset. I think they are in all of the generate functions now.